Soccar [Game-Server]/src/nl/soccar/gameserver/model/command/listener/MuteCommandListener.java

```diff
@@ -7,10 +7,15 @@
 import nl.soccar.library.enumeration.Privilege;
 
 /**
+ * Listener for mute commands.
+ * 
  * @author Lesley
  */
 public final class MuteCommandListener extends CommandListener {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)